### PR TITLE
[libxml2] update to 2.10.3

### PR DESCRIPTION
**Pull request by Neumann-A**

### CODE REVIEW

File: ports/libxml2/vcpkg.json [hunk 1]
```diff
@@ -1,7 +1,6 @@
 {
   "name": "libxml2",
-  "version": "2.9.14",
-  "port-version": 1,
+  "version": "2.10.3",
   "description": "Libxml2 is the XML C parser and toolkit developed for the Gnome project (but usable outside of the Gnome platform).",
   "homepage": "https://xmlsoft.org/",
```
Comment (dg0yt) [inline ports/libxml2/vcpkg.json:5]: Please change this link. It has been defect/unsafe for a while. It is one of the vcpkg "problems" on repology.org:
https://repology.org/repository/vcpkg/problems
I suggest https://gitlab.gnome.org/GNOME/libxml2/-/wikis/home

**Comment by Neumann-A** [comment]
@dg0yt How was the option to add additional log files named? I am somehow unable to find it and ffmpeg fails pkg-config for libxml2 for some strange reason.

**Comment by dg0yt** [comment]
@Neumann-A SAVE_LOG_FILES : https://github.com/microsoft/vcpkg/blob/master/docs/maintainers/vcpkg_execute_required_process.md#save_log_files

**Comment by Neumann-A** [comment]
ok the reason i couldn't find it was because my checkout was an old local branch with the same name

File: ports/ffmpeg/portfile.cmake [hunk 2]
```diff
@@ -540,6 +540,7 @@ if (NOT VCPKG_BUILD_TYPE OR VCPKG_BUILD_TYPE STREQUAL "release")
         COMMAND "${SHELL}" ./build.sh
         WORKING_DIRECTORY "${BUILD_DIR}"
         LOGNAME "build-${TARGET_TRIPLET}-rel"
+        SAVE_LOG_FILES ffbuild/config.log
```
Comment (dg0yt) [inline ports/ffmpeg/portfile.cmake:543]: If you want to keept this, remember to rename the first saved log file before starting the second run.

**Review by MonicaLiu0311 — APPROVED** [review]

**Pull request merged**